src/services/secrets-service.js

```diff
@@ -19,11 +19,11 @@ class SecretsDb {
     this.debug = createDebug('actual:secrets-db');
     this.db = null;
     this.initialize();
-    this.migrateNordigen();
+    this.migrateGoCardless();
   }
 
-  /// Migrates nordigen from config.json or process.env to app secret
-  migrateNordigen() {
+  /// Migrates GoCardless from config.json or process.env to app secret
+  migrateGoCardless() {
     if (
       config.nordigen &&
       (!this.get(SecretName.nordigen_secretId) ||
```

Comment:
What do you want to do about the secret names? Would it make sense to migrate them to `gocardless_*`?

Reply:
Yea, I'd like to do this, but separately. It's a non-trivial operation to do this with backwards compatibility both in the UI and API layer.